.ruff.toml

```
@@ -85,6 +86,11 @@ ignore = [
     "FBT002",  # boolean-default-value-in-function-definition
     "FBT003",  # boolean-positional-value-in-function-call
 
+    # flake8-fixme (FIX)
+    "FIX001",  # Line contains FIXME.  TODO: all FIXME should be fixed and no new ones added!
```

```suggestion
    "FIX001",  # Line contains FIXME.  this should be fixed or at least FIXME replaced with TODO 
```
Actually fixing all the `FIXME` occurrences is a good long term goal, but might be challenging. Replacing `FIXME` with `TODO` is a small but quick and low-effort improvement because it makes it simpler to find problematic code in the future, whether by grepping or by using Ruff.